apps/chat/bots.py

```diff
@@ -91,7 +92,13 @@ def _initialize(self):
         if self.child_chains and not self.default_child_chain:
             self.default_tag, self.default_child_chain = list(self.child_chains.items())[0]
 
-        self.chain = create_experiment_runnable(self.experiment, self.session)
+        self.chain = create_experiment_runnable(self.experiment, self.session, self.disable_tools)
+
+        terminal_route = (
+            ExperimentRoute.objects.select_related("child").filter(parent=self.experiment, type="terminal").first()
+        )
+        if terminal_route:
+            self.terminal_chain = create_experiment_runnable(terminal_route.child, self.session)
```

Comment:
not sure if this is just the diff view but seems like a merge duplicate

Reply:
Ooh, good catch. Hmm, this makes me wonder if there aren't any more of these. I'll have to scrutinize a bit